apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -253,7 +253,7 @@ export class PublishAction extends BaseRushAction {
         for (const change of orderedChanges) {
           if (change.changeType && change.changeType > ChangeType.dependency) {
             const project: RushConfigurationProject | undefined = allPackages.get(change.packageName);
-            if (project) {
+            if (project && !this._packageExists(project)) {
```

Comment:
>project [](start = 16, length = 7)

If every single package that we're trying to publish already exists, maybe that should be considered an error.